django/test/selenium.py

```diff
@@ -87,14 +87,15 @@ def create_options(self):
         return options
 
     def create_webdriver(self):
+        options = self.create_options()
         if self.selenium_hub:
             from selenium import webdriver
 
-            return webdriver.Remote(
-                command_executor=self.selenium_hub,
-                desired_capabilities=self.get_capability(self.browser),
-            )
-        return self.import_webdriver(self.browser)(options=self.create_options())
+            for key, value in self.get_capability(self.browser).items():
+                options.set_capability(key, value)
```

Comment:
`set_capability()` was added in selenium 3.14.1, however, `selenium` < 4.8 doesn't support Python 3.10 which is the minimum version supported by Django 5.0+ , see #17013.